js/wp-seo.js

```diff
@@ -143,4 +147,45 @@
 		} );
 	} );
 
+	if ( images ) {
+		images
+			.each(function ( index, image ) {
+				var imageblock = jQuery( image );
+				var addImgLink = imageblock.find( '.upload-custom-img' ),
+				delImgLink = imageblock.find( '.delete-custom-img' ),
+				imgContainer = imageblock.find( '.custom-img-container' ),
+				imgIdInput = imageblock.find( '.custom-img-id' );
+				delImgLink
+					.on( 'click', function( e ){
```

Comment:
I like using `evt` (or `err` when applicable) instead of `e`
 in callbacks to distinguish between `Event` and `Error`